vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -406,233 +399,26 @@ winrt::Microsoft::ReactNative::JSValue TouchEventHandler::GetPointerJson(const R
       {"altKey", pointer.altKey}};
 }
 
-#ifdef USE_FABRIC
-// This work should probably be delegated to the ComponentViews
-facebook::react::SharedEventEmitter EventEmitterForElement(
```

Comment:
is the idea that all of this "if fabric" moves over to another file, or where is it going?

Reply:
Yeah, fabric will have an alternative version of this file - parts of this logic are in shared fabric code - others will be quite different from the xaml code.